pom.xml

```diff
@@ -41,6 +41,12 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>nz.net.ultraq.thymeleaf</groupId>
+            <artifactId>thymeleaf-layout-dialect</artifactId>
+            <version>2.0.5</version>
```

Comment:
Jeżeli pozbędziesz się atrybutu <version> i pozwolisz żeby wersja została zaciągnięta z zależności spring boota, to klasa konfiguracji ThymeleafConfig nie będzie Ci już potrzebna